server/utils/firebase_algolia_server_code/functions/test/expectedMockOutputHelpers.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Description: These functions can be used to verify that the functions under
+ *              test return whatever data the algoliaOperation they are called
+ *              with returns. I.e., if a function is supposed to return the 
+ *              the result of a saveObject operation on the input data, put
+ *              the input data through the corresponding getFormattedX
+ *              and assert.deepEqual(output, formatted(input)).
+ * Author: Robert Marcus
+ * Date: July 7, 2020
+ */
+
+const { addField } = require('./algoliaMocks');
+
+/**
+ * Returns a snap id object as it would be by an algolia delete mock
+ * @param {Object} snap a firebase snap object
+ * @return {Object}
+ */
+function getExpectedOutputID(snap) {
+  return addField({'objectID': snap.id});
+}
+
+/**
```

Comment:
Move this to explicitly be in every test.  Otherwise people have to jump into this file to find out what your test is expecting.

Reply:
This is a deleted file. Think you are seeing it by accident.